source/virtualBuffers/adobeAcrobat.py

```diff
@@ -120,3 +121,10 @@ def event_valueChange(self, obj, nextHandler):
 			return nextHandler()
 		if not self._handleScrollTo(obj):
 			return nextHandler()
+
+	def _get_ElementsListDialog(self):
+		return ElementsListDialog
+
+class ElementsListDialog(browseMode.ElementsListDialog):
+
+	ELEMENT_TYPES=(browseMode.ElementsListDialog.ELEMENT_TYPES[0],browseMode.ElementsListDialog.ELEMENT_TYPES[1])

```

Comment:
Ah,  the winword one is a little different as that is also adding extra 
items to the tuple at the same time. Python 2 has no syntax for that. In 
Python 3 I think you can write new = (*old[0:2],blah,blah)




On 30/06/2016 3:55 PM, Leonard de Ruijter wrote:
>
> In source/virtualBuffers/adobeAcrobat.py 
> <https://github.com/nvaccess/nvda/pull/6118#discussion_r69075184>:
>
> > @@ -120,3 +121,10 @@ def event_valueChange(self, obj, nextHandler):
> >  			return nextHandler()
> >  		if not self._handleScrollTo(obj):
> >  			return nextHandler()
> > +
> > +	def _get_ElementsListDialog(self):
> > +		return ElementsListDialog
> > +
> > +class ElementsListDialog(browseMode.ElementsListDialog):
> > +
> > +	ELEMENT_TYPES=(browseMode.ElementsListDialog.ELEMENT_TYPES[0],browseMode.ElementsListDialog.ELEMENT_TYPES[1])
> I agree, although the approach I used also is used in 
> NVDAObjects.window.winword.ElementsListDialog. If you like, I can 
> apply your feedback there as well for this request.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/6118/files/9c845bfa0a5d6e06da86a55b3d9bedb4b07d6a98#r69075184>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANf5nTk-gIL007jxp0SnLURvw1S2Ar-xks5qQ1pPgaJpZM4I__Yv>.
>

-- 
Michael Curran
Executive Director, NV Access Limited
Phone: +61 7 3149 3306
Website: http://www.nvaccess.org/
Twitter: @NVAccess
Facebook: http://www.facebook.com/NVAccess